### DISCUSSION TITLE
FEATURE REQUEST Inverting of controls

### DISCUSSION THREAD

**Discussion by MorbidMorke** [howroyd/simonsays#39] [Ideas]
I would love it if it's possible to do some sort of magical wizardry and make it so that with a specific command (perhaps invert) it would make forward (w) read as backward (s) or left (a) read as right (d). 

Then it would invert the controls for maybe 3-5 seconds if that's possible.

**Comment by howroyd** [comment]
This would require complete hijacking of the keyboard and mouse as we'd be blocking windows from seeing the drive and replacing it with our own.

It's a lot of work but do-able.  Need to think about the security of this too.
